pkgs/applications/science/misc/root/default.nix

```
@@ -55,31 +58,21 @@
 , noSplash ? false
 }:
 
-let
-
-  _llvm_9 = llvm_9.overrideAttrs (prev: {
-    patches = (prev.patches or [ ]) ++ [
-      (fetchpatch {
-        url = "https://github.com/root-project/root/commit/a9c961cf4613ff1f0ea50f188e4a4b0eb749b17d.diff";
-        stripLen = 3;
-        hash = "sha256-LH2RipJICEDWOr7JzX5s0QiUhEwXNMFEJihYKy9qWpo=";
-      })
-    ];
-  });
-
-in
-
 stdenv.mkDerivation rec {
+VERBOSE=1;
+failureHook = ''
+  find . -name rootcling_stage1 -exec ldd {} \;
+'';
```

It'll look better with unified indentation.

```suggestion
  VERBOSE=1;
  failureHook = ''
    find . -name rootcling_stage1 -exec ldd {} \;
  '';
```

---

That was for debugging the issue with builtin_llvm=OFF. I will be reverting this later.